### PR TITLE
docs(*) add list of annotations used in k8s

### PR DESCRIPTION
**Pull request by bartsmykla**
image

### CODE REVIEW

File: docs/docs/1.2.1/documentation/overview.md [hunk 1]
```diff
@@ -194,6 +194,269 @@ spec:
...
+kind: Pod
+metadata:
+  name: example
+  annotations:
+    prometheus.metrics.kuma.io/port: "1234"
+```
+
+#### `prometheus.metrics.kuma.io/path`
+
+Metrics prometheus path to override `Mesh`-wide default path where prometheus should scrape metrics from
```
Comment (jpeach) [inline docs/docs/1.2.1/documentation/overview.md:386]:
```suggestion
This annotation overrides the `Mesh`-wide default path where Prometheus uses to scrape metrics.
```

File: docs/docs/1.2.1/documentation/overview.md [hunk 2]
```diff
@@ -194,6 +194,269 @@ spec:
...
+kind: Pod
+metadata:
+  name: example
+  annotations:
+    prometheus.metrics.kuma.io/path: "/custom-metrics"
+```
+
+#### `kuma.io/builtindns`
+
+Instruct sidecar to use its builtin DNS server
```
Comment (jpeach) [inline docs/docs/1.2.1/documentation/overview.md:401]:
```suggestion
Instructs the Kuma sidecar to use its builtin DNS server.
```

**Review by jpeach — CHANGES_REQUESTED** [review]
This is great! I made some trivial copy editing suggestions.

**Comment by Bradamant3** [comment]
Hey Bart, I did a pretty drastic thing here and moved the whole annotations section to its own page, under a new left nav heading "Reference docs" where it now lives with the HTTP API reference. There's a link from the original page where it lived, too.

See 
https://deploy-preview-478--kuma.netlify.app/docs/1.2.1/documentation/kubernetes-annotations/
https://deploy-preview-478--kuma.netlify.app/docs/1.2.1/documentation/overview/#kubernetes-mode (scroll down a bit for the link to the new annotations page)

let me know what you think!

**Pull request merged**